buildenv/jenkins/docker-slaves/ppc64le/centos7/Dockerfile

```diff
@@ -110,27 +109,10 @@ RUN rm /usr/bin/c++ /usr/bin/cc \
   && ln -s gcc /usr/bin/cc
 
 # Install GCC-7.3.1
-RUN yum -y update \
-  && yum -y install environment-modules \
-  && yum clean all \
-  && source /etc/profile.d/modules.sh \
-  && wget ftp://ftp.unicamp.br/pub/linuxpatch/toolchain/at/redhat/RHEL7/gpg-pubkey-6976a827-5164221b \
-  && rpm --import gpg-pubkey-6976a827-5164221b \
-  && echo "# Begin of configuration file" >> /etc/yum.repos.d/at11.0.repo \
-  && echo "[at11.0]" >> /etc/yum.repos.d/at11.0.repo \
-  && echo "name=Advance Toolchain Unicamp FTP" >> /etc/yum.repos.d/at11.0.repo \
-  && echo "baseurl=ftp://ftp.unicamp.br/pub/linuxpatch/toolchain/at/redhat/RHEL7" >> /etc/yum.repos.d/at11.0.repo \
-  && echo "failovermethod=priority" >> /etc/yum.repos.d/at11.0.repo \
-  && echo "enabled=1" >> /etc/yum.repos.d/at11.0.repo \
-  && echo "gpgcheck=1" >> /etc/yum.repos.d/at11.0.repo \
-  && echo "gpgkey=ftp://ftp.unicamp.br/pub/linuxpatch/toolchain/at/redhat/RHEL7/gpg-pubkey-6976a827-5164221b" >> /etc/yum.repos.d/at11.0.repo \
-  && echo "# End of configuration file" >> /etc/yum.repos.d/at11.0.repo \
-  && yum -y update \
-  && yum -y install \
-    advance-toolchain-at11.0-runtime \
-    advance-toolchain-at11.0-devel \
-    advance-toolchain-at11.0-perf \
-  && yum clean all
+RUN cd /usr/local \
+  && wget -O gcc-7.tar.xz "https://ci.adoptopenjdk.net/userContent/gcc/gcc730+ccache.ppc64le.tar.xz" \
```

Comment:
@jdekonin or @vsebe are you good with this change?

Reply:
lgtm